smart-contracts/contracts/dex-router-osmosis/.cargo/config.toml

```diff
@@ -2,5 +2,5 @@
 wasm = "build --release --target wasm32-unknown-unknown"
 unit-test = "test --lib"
 schema = "run --bin schema"
-test-tube = "test --package dex-router-osmosis --lib -- --include-ignored test_tube:: --nocapture --test-threads=1"
```

Comment:
`--test-threads=1`  is needed due to some race condition that was causing not deterministic test outcomes.

Reply:
added back